datashare-app/src/main/java/org/icij/datashare/web/ProjectResource.java

```diff
@@ -201,13 +208,21 @@ public Payload deleteProject(String id, Context context) throws IOException {
         @Delete("/")
         public Payload deleteProjects(Context context) {
             DatashareUser user = (DatashareUser) context.currentUser();
+            Logger logger = LoggerFactory.getLogger(getClass());
             getUserProjects(user).forEach(project -> {
                 try {
                     deleteProject(project.name, context);
                 } catch (IOException e) {
                     throw new RuntimeException(e);
                 }
             });
+            try {
+                logger.info("Stopping tasks : {}", taskManager.stopAllTasks(user));
+                taskManager.waitTasksToBeDone(TaskManager.POLLING_INTERVAL*2, MILLISECONDS);
+                logger.info("Deleted tasks : {}", !taskManager.clearDoneTasks().isEmpty());
+            } catch (IOException e) {
```

Comment:
I'd let the IOException be thrown directly instead of wrapping it in a RuntimeException but maybe I'm missing something?